specification/gedcom.md

```diff
@@ -516,6 +493,125 @@ An extension tag that is not given a URI in the schema structure is called an **
 The meaning of an undocumented extension tag is identified by its tag.
 
 
+### Types of Extensions
+
+Inside a date payload, an extension tag may be used where the specification documents a calendar;
+if an extension calendar is used, extensions may also be used for the month and epoch.
+If a standard tag exists for a given calendar, month, or epoch, it should be used instead of an extension tag, but using an extension tag instead is permitted to enable forward compatibility with versions of this standard that add additional calendars.
+
+Inside an enumeration or list-of-enumeration payload, extension tags may be used to extend the set of permitted enumerated values in that context.
+Documented extension tags with URIs of enumerated values in this document may be used and permit using values not normally permitted as payloads of their containing structure.
+Use of a documented extension tag to indicate a structure type, calendar, or month as an enumerated value is also permitted, as for example is done by [SOUR.EVEN](#enum-SOUR.EVEN) and [NO](#enum-NO) in this specification.
+
+:::example
+The following is not allowed because `OTHER` is not defined as a value for `ADOP`:
+
+```gedcom
+0 @BAD@ INDI
+1 FAMC @F1@
+2 ADOP OTHER
+```
+
+However, if the following had appeared in the header:
+
+```gedcom
+1 SCHMA
+2 TAG _OTHER https://gedcom.io/terms/v7/enum-OTHER
+```
+
+then `_OTHER` would be permitted as a payload of `ADOP` (with the meaning `OTHER` has in this specification: "A value not listed here; should have a `PHRASE` substructure")
+
+```gedcom
+0 @GOOD@ INDI
+1 FAMC @F1@
+2 ADOP _OTHER
+3 PHRASE As an heir of the estate
+```
+:::
+
+
+A **relocated standard structure** is a structure whose tag is a documented extension tag with a URI that identifies it as a structure type in this specification.
+Regardless of its structure type, a relocated standard structure may appear as a record, in the header, or with any structure type as its superstructure.
+It must abide by all of the other restrictions of its structure type.
+
+> Note: make it clearer about records and so on
```

Comment:
Not sure what this means.  Is this a TODO that is intended to be addressed before merging this PR?

Reply:
Addressed in fdcaf4c